src/evaluations/evaluations.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Post, Put, Delete, Param, Body, UseGuards } from '@nestjs/common';
+import { EvaluationDto } from './dto/evaluation.dto';
+import { CreateEvaluationDto } from './dto/create-evaluation.dto';
+import { UpdateEvaluationDto } from './dto/update-evaluation.dto';
+import { EvaluationsService } from './evaluations.service';
+import { JwtAuthGuard } from '../guards/jwt-auth.guard';
+
+@Controller('evaluations')
+export class EvaluationsController {
```

Comment:
Please look up in the nestjs documentation the way to configure guards at the top level of a controller so you do not have to repeat the `@UseGuards` decorator for each func.